config/environments/production.rb

```diff
@@ -6,14 +6,6 @@
   config.action_controller.perform_caching = true
   config.force_ssl = true
 
-  config.asset_host = proc do |_source, request|
-    # we want precompiled assets to have domain-agnostic URLs
-    # and request is nil during asset precompilation
-    (IdentityConfig.store.asset_host.presence || IdentityConfig.store.domain_name) if request
-  end
-  config.assets.compile = false
-  config.assets.digest = true
-  config.assets.gzip = false
```

Comment:
do these options no longer exist? or are they the new defaults?

Reply:
They no longer exist.

I also came across them when looking through the Propshaft source, commented as being deprecated:

https://github.com/rails/propshaft/blob/69c285f/lib/propshaft/railtie.rb#L69-L74